src/main/java/org/jenkinsci/plugins/ghprb/GhprbTrigger.java

```diff
@@ -70,7 +72,7 @@
     private static final Logger logger = Logger.getLogger(GhprbTrigger.class.getName());
 
     private static final ExecutorService pool = Executors.newFixedThreadPool(
```

Comment:
I meant did you test what this change will do for you?

It totally makes sense to make this configurable, just arguing that defaulting to the amount of cores could lead to some very large threadpools :)